src/components/sidebar/index.tsx

```diff
@@ -53,10 +71,7 @@ const Sidebar = () => {
             key={i}
             text={list[key as unknown as number]}
             active={i === active}
-            onClick={() => {
-              setActive(i)
-              navigate(`#${list[key as unknown as number]}`)
-            }}
+            onClick={() => navigate(`#${list[key as unknown as number]}`)}
```

Comment:
@Edqe14 there's no reason to use this navigate function, I've confirmed that a normal link works perfectly fine.